templates/registration/logged_out.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
Please review the checklist for the task again. You don't extend base.html, and you also don't have a button that allows us to log in again.